lib/rack/test.rb

```diff
@@ -30,7 +30,14 @@ module Rack
   module Test
     # The default host to use for requests, when a full URI is not
     # provided.
-    DEFAULT_HOST = 'example.org'.freeze
+    def self.default_host
```

Comment:
We could maybe leave in the constant DEFAULT_HOST for people using a work around?